src/internal/pfsdb/model.go

```diff
@@ -84,11 +85,23 @@ func parseBranches(branchInfos string) ([]*pfs.Branch, error) {
 	return branches, nil
 }
 
-// Commit is a row in the pfs.commits table.
 type Commit struct {
-	ID          CommitID `db:"id"`
-	Repo        Repo     `db:"repo"`
-	CommitSetID string   `db:"commit_set_id"`
+	ID             CommitID  `db:"int_id"`
+	CommitSetID    string    `db:"commit_set_id"`
```

Comment:
Probably not the right place to fix this, but CommitSetID should refer to another commit, not a string right?